articles/digital-assistants-vs-digital-companions/index.md

```diff
@@ -0,0 +1,63 @@
+
+We use digital assistants in various forms, like in our smartphones, speakers, etcetera. Most of us have a fair share of interaction with digital assistants in various shapes and forms. The interaction of digital assistant takes place through hands-free [Voice User Interfaces](https://www.interaction-design.org/literature/topics/voice-user-interfaces) (a system that allows users to interact through voice or speech commands) and text or visuals. Examples of digital assistants we use today are plus [Siri](https://www.apple.com/siri/), [Alexa](https://www.cnet.com/news/what-is-alexa/), and Google.
+
+Digital assistants are affordable, convenient, and user-friendly and are primarily used to save time and carry commands. They respond to our requests reactively rather than engaging us proactively.
+
+### An overview of digital companions
+
+Digital companions are just new and improved digital assistants. In a way, they assist us in numerous tasks, and they do so in a personalized, seamless, and fun manner. They are interactive to the user. Digital companions learn and remember what we like or dislike each time we interact, that is individual preferences and needs. They also assist us in understanding and choosing, among many options, the best course of action. In other words, digital companions interact with us proactively as opposed to digital assistants, which do so reactively.
+
+Digital companions have a two-way relationship with their users based on empathy, trust, and anticipation of needs and preferences. They make suggestions that fit our interests and positively influence our behaviors.
+
+Digital companions are used to automating management and employee experience, thereby making it more accessible. It is also used in performing common tasks such as resetting passwords, system status updates, and outage alerts.
+
+### A comparison of digital assistants and digital companions
+
+- Digital assistants&#39; interactions are primarily command-based, but those of digital companions are both proactive and command-based.
+- Digital assistants create short and one-sided exchanges; digital companions create more extended and bidirectional exchanges.
+- Digital assistants use a single primary modality, for instance, voice or text; digital companions use multiple modalities, e.g., voice, movements.
+- Digital assistants are more of generic character and less personality; digital companions are personalized for each user.
+- Digital assistants&#39; experiences are homogenous for all users; digital companions&#39; experiences are personalized for each individual.
+- Digital assistants&#39; actions are executed based on users&#39; commands; digital companions&#39; actions are executed based on context and goals.
+- Digital assistants incorporate very little information about the user while digital companions remember users` needs and preferences.
```

Comment:
Don't use \` in markdown. Even if you want to use \` in your article, use \\` to represent the character. Moreover, ` is used for code formatting and variables highlighting in markdown. Let me know if you have any doubts regarding this.